content/3_Silver/DFS.mdx

```diff
@@ -585,3 +585,35 @@ public class BuildingTeams
 ### Problems
 
 <Problems problems="bip" hideSuggestProblemButton />
+
+## Quiz
+<Quiz>
+	<Quiz.Question>
+		Can a DFS with adjacency list be used to find shortest paths in linear time?
+		<Quiz.Answer>
+			Yes
+			<Quiz.Explanation>
+				Incorrect. Worst case, a DFS shortest path algorithm would have to iterate through <i>every</i> path before finding the shortest. Working shortest path algorithms will be covered later in the Gold section.
```

Comment:
No need to duplicate stuff between answers considering that all of them are shown.